scram/route_manager/api/views.py

```diff
@@ -166,13 +172,13 @@ def find_entries(arg, active_filter=None):
         try:
             pk = int(arg)
             query = Q(pk=pk)
-        except ValueError:
+        except ValueError as exc:
```

Comment:
What is exc?

Reply:
This and a few others were to address this check: https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/

I just followed the naming there (it's the exception).